bin.src/make_job_document.py

```diff
@@ -1,4 +1,25 @@
 #!/usr/bin/env python
+# This file is part of faro.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
```

Comment:
Don't forget to also add this COPYRIGHT file.

Reply:
Thanks Tim, comment resolved in commit bd66ab0